shell.nix

```diff
@@ -127,6 +127,16 @@ let
     STACK_IN_NIX_SHELL = "true";
   };
 
+  # A shell which provides env for the demo application
+  demoShell = pkgs.mkShell {
+    name = "hydra-demo-shell";
+    buildInputs = [
+      cardano-node.cardano-node
+      cardano-node.cardano-cli
+      hsPkgs.hydra-node.components.exes.hydra-node
```

Comment:
Have you used this for running the whole demo? It likely will not have `hydra-tui` in scope, right? So we might want to add this one here as well.

Reply:
Good catch. That is why I left it as a draft - I'm going to run the demo and then turn it into a proper PR.